patroni/tags.py

```diff
@@ -45,8 +47,29 @@ def clonefrom(self) -> bool:
 
     @property
     def nofailover(self) -> bool:
-        """``True`` if ``nofailover`` is ``True``, else ``False``."""
-        return bool(self.tags.get('nofailover', False))
+        """Common logic for obtaining the value of ``nofailover`` from ``tags`` if defined.
+
+        If ``no_failover is not defined, this methods returns True if ``failover_priority`` in
+        :attr:`Member`.tags`` is non-positive, else defaults to ``False``.
```

Comment:
```suggestion
        If ``nofailover`` is not defined, this methods returns ``True`` if ``failover_priority`` is non-positive,
        ``False`` otherwise.
```